shell/initialize/client.js

```diff
@@ -322,7 +322,7 @@ async function mountApp(__app) {
     });
 
     // Push the path and let route to be resolved
-    router.push(path, undefined, (err) => {
+    router.push(path, () => {}, (err) => {
```

Comment:
I explained this above. This allowed us to remove the router.push override that nuxt did.